src/components/CardMenus/CommonCard.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+// eslint-disable-next-line import/order
+// eslint-disable-next-line import/order
+import { Card, CardMedia, CardContent, CardHeader, Typography, Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+// import Avatar from '@material-ui/core/Avatar';
+// eslint-disable-next-line import/order
+
+// eslint-disable-next-line import/order
+// eslint-disable-next-line import/order
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    maxWidth: 345,
+    width: '72%',
+    marginLeft: '250px',
+  },
+  media: {
+    height: 0,
+
+    paddingTop: '56.25%', // 16:9
+  },
+  content: {
+    padding: '0px',
+    marginLeft: '18px',
+  },
+  header: {
+    padding: '0px',
+    marginLeft: '18px',
+  },
+  expand: {
+    transform: 'rotate(0deg)',
+    marginLeft: 'auto',
+    transition: theme.transitions.create('transform', {
+      duration: theme.transitions.duration.shortest,
+    }),
+  },
+  expandOpen: {
+    transform: 'rotate(180deg)',
+  },
+  avatar: {
+    backgroundColor: 'red',
+  },
+  image: {
+    padding: '30px',
+    border: '10px',
+    borderRadius: '200px',
+  },
+}));
+const CommonCard = ({ name, price, resturantName, img }) => {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <Card className={classes.root}>
+        <Box className={classes.image}>
+          <CardMedia className={classes.media} image={img} title={name} />
+        </Box>
+        <CardHeader
+          className={classes.header}
+          // eslint-disable-next-line react/jsx-no-duplicate-props
```

Comment:
Why are you using eslint disable for duplicate props when there isn't a duplicate prop.